app/scripts/modules/core/src/deploymentStrategy/strategies/redblack/redblack.strategy.ts

```diff
@@ -28,7 +28,7 @@ DeploymentStrategyRegistry.registerStrategy({
       rollback: {
         onFailure: false,
       },
-      maxRemainingAsgs: !command.strategy ? 2 : command.maxRemainingAsgs,
+      maxRemainingAsgs: command.maxRemainingAsgs !== undefined ? command.maxRemainingAsgs : 2,
```

Comment:
wondering if you're intentionally excluding `null` check, or if we can do:

```suggestion
      maxRemainingAsgs: command.maxRemainingAsgs ?? 2,
```

Reply:
Oh right! That's a thing we can do now! Nope, not intentional.